.circleci/config.yml

```diff
@@ -26,9 +26,9 @@ jobs:
           name: CI-Build
           command: npm run cibuild
 
-  install-and-cibuild: # main cibuild using node 16 & npm 7
+  install-and-cibuild: # main cibuild using node 18 & npm 10
     docker:
-      - image: cimg/node:16.17.1
+      - image: cimg/node:18.20
```

Comment:
Thanks for the note and PR. 
Below there are more `node:16` dockers in this config.
Also I prefer to use specific versions for the tests.
So I may suggest you include the patch version number e.g. `18.20.0`?

Reply:
Oh, didn't notice. I've set them all to the 18.20.4 that they list here: https://circleci.com/developer/images/image/cimg/node